functionary/core/models/workflow.py

```diff
@@ -68,4 +79,13 @@ def ordered_steps(self):
     def parameters(self):
         """Convenience alias for workflowparameter_set"""
         # Provides better static type checking than using related_name
-        return self.workflowparameter_set
+        return self.workflowparameter_set  # type: ignore
+
+    def deactivate(self):
+        """Deactivate the workflow and pause any associated scheduled tasks"""
+        self.active = False
+        self.save()
```

Comment:
Yes, it would.  If someone wanted to just change the active state without saving yet, they could just do that by setting `workflow.active = False` and then doing some other stuff before saving.